app/policies/va_profile_policy.rb

```diff
@@ -8,7 +8,7 @@ def access?
   def access_to_v2?
     if Flipper.enabled?(:va_v3_contact_information_service, user)
       # user vet360_id is no longer needed for Contact Information Api V2
-      user.vet360_id.present? || user.idme_uuid.present? || user.logingov_uuid.present?
+      user.vet360_id.present? || user.icn.present? || user.idme_uuid.present? || user.logingov_uuid.present?
```

Comment:
This allows users to find info with any uuid

Reply:
I was looking at the original requirements to contacting the VA Profile service. It used to require a verified user, is that still the case?

If so, we probably shouldn't have an OR for idme_uuid/logingov_uuid, as that allows access to every single user on va.gov, not necessarily only verified users

Reply:
Additionally, every verified user will have ICN. It seems like things could be simplified quite a bit if the service was only ever called with ICN.